models.py

```diff
@@ -343,7 +343,7 @@ def unproofed_galleys(self, galleys):
     def time_to_due(self):
         due = self.due - timezone.now()
 
-        if due.days == 0:
+        if due.seconds < 86400:
```

Comment:
24 * 60 * 60 might be easier to read but I think the context is self explanatory